documentation/modules/rn-2.5.adoc

```diff
@@ -25,30 +25,28 @@ This release has the following technical changes:
 In this version of {project-short}, migration using OpenStack source providers graduated from a Technology Preview feature to a fully supported feature.
 
 .Disabling FIPS
-
-EMS enforcement is disabled for migrations with VMware vSphere source providers to enable migrations from versions of vSphere that are supported by {project-short} but do not comply with the 2023 FIPS requirements.
+g
+{project-short} 2.5.6 enables migrations from vSphere source providers by not enforcing Enterprise Master Secret (EMS). This enables migrating from all vSphere versions that {project-short} supports, including migrations that do not meet 2023 FIPS requirements.
```

Comment:
this change was introduced before 2.5.6, we shouldn't mentioned 2.5.6 here

Reply:
Removed 2.5.6